.github/workflows/e2e.yml

```diff
@@ -159,11 +159,17 @@ jobs:
     - name: Make test
       run: make test-short-ci CSI_VERSION=$CSI_VERSION OPERATOR_VERSION=$CSI_OPERATOR_VERSION CHARTS_DIR=${{env.CSI_BAREMETAL_OPERATOR_DIR}}/charts
 
+    - name: Publish Unit Test Results
+      uses: EnricoMi/publish-unit-test-result-action@v1
+      if: always()
```

Comment:
Do we need to check presence of report before uploading it?

Reply:
The if: always() clause guarantees that this action always runs, even if earlier steps (e.g., the unit test step) in your workflow fail.